src/infra/runtime/runtimeQueryRepo.go

```diff
@@ -295,3 +297,17 @@ func (repo RuntimeQueryRepo) ReadPhpConfigs(
 		phpModules,
 	), nil
 }
+
+func (repo RuntimeQueryRepo) IsHtaccessModifiedRecently() bool {
```

Comment:
That's just wrong man. Htaccess is a tech name. If you really needed to call that on the domain, which you don't, you should have used RewriteFile or something more generic. We don't control .htaccess by the domain. That's a OLS/Apache particularity, it's not a business rule.